Assets/Scripts/Bullet.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
```

Comment:
To be discussed in a call => Shouldn't have hard-coded damage variable, so implementation details can be instead implemented as default behavior in a Damaging Projectile interface

Reply:
Also for the projectile interface, [check out this reference to require the collider component](https://docs.unity3d.com/ScriptReference/RequireComponent.html)